ENGAGEHF/Dashboard/Vitals/Views/VitalsCard.swift

```diff
@@ -0,0 +1,109 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import HealthKit
+import SwiftUI
+
+
+struct VitalsCard: View {
+    @ScaledMetric private var quantityTextSize: CGFloat = 20
+    
+    let quantity: String?
+    let units: String?
+    let type: String?
+    let date: Date?
+    
+    private var displayDate: String? {
+        date?.formatted(date: .numeric, time: .omitted)
```

Comment:
I think it would make sense to also show the time here, adds more context especially if we decrease the size of the date display text.
```suggestion
        date?.formatted(date: .numeric, time: .shortened)
```